tests/Unit/Extension/Propagator/B3/B3PropagatorTest.php

```diff
@@ -20,11 +20,10 @@
  */
 class B3PropagatorTest extends TestCase
 {
-    private const B3_SINGLE_TRACE_ID_BASE16 = 'ff0000000000051791e0000000000041';
-    private const B3_SINGLE_SPAN_ID_BASE16 = 'ff00051791e00041';
-    private const B3_SINGLE_HEADER_SAMPLED = self::B3_SINGLE_TRACE_ID_BASE16 . '-' . self::B3_SINGLE_SPAN_ID_BASE16 . '-1';
-    private const B3_MULTI_TRACE_ID_BASE16 = 'ff000000000000301710000000000041';
-    private const B3_MULTI_SPAN_ID_BASE16 = 'ff00003017100041';
+    private const B3_TRACE_ID_BASE8 = 'ff00051791e00041';
```

Comment:
Confusing/incorrect name, `BASE16` was likely referring to hex encoding (otherwise `B3_SPAN_ID_BASE16` should be renamed).